indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/Main.scala

```diff
@@ -49,7 +49,11 @@ object Main extends WellcomeTypesafeApp {
 
     indexCreator.create(
       index = index,
-      mappingDefinition = BagsIndexConfig.mapping
+      mappingDefinition = BagsIndexConfig.mapping,
+      settings = Map(
+        // The largest number of files on a bag is ~ 970,000
```

Comment:
Let's put some more clues for a future reader in this comment:

```suggestion
        // The largest number of files on a bag is ~ 970,000 (see b19974760, aka
        // Chemist and Druggist)
        //
        // If we use the default limit, we get errors from Elasticsearch:
        //
        //      [illegal_argument_exception] The length of [files.path] field of
        //      [digitised/b19974760] doc of [storage_bags] index has exceeded [1000000] -
        //      maximum allowed to be analyzed for highlighting. This maximum can be
        //      set by changing the [index.highlight.max_analyzed_offset] index level setting.
        //      For large texts, indexing with offsets or term vectors is recommended!
        //
```

Reply:
that's actually the wrong error you've quoted now I look at it

Reply:
Well what's the right error? :)